pythonExecSource/docs/Usage.md

```diff
@@ -0,0 +1,302 @@
+
+In addition, any values specified in the `presetValues` parameter will be set as global variables.
+
+#### Receiving Data from the Source
+
+Although probably unusual, the Python code being executed can send events to the associated Vantiq source. This is done using the `send_notification()` method in the Vantiq Connector SDK. This is a method associate with the `connector_connection` global variable outlined above.  To make such a call, use
+
+```python
+    await connector_connection.send_notification({'id': str(uuid.uuid4())})
+```
+
+That will send an event from the connector's source containing the values specified.
+
+On the Vantiq side, assuming our source is named `pythonSource`, we would define a rule as follows:
+
+```
+RULE pythonSourceEvebt
+WHEN EVENT OCCURS ON "/sources/pythonSource" as sourceEvent
+
+// Grab the actual value from the event
+var message = sourceEvent.value
+
+log.info("Got a message: {}", [message])
+
+```
+
+### Deploying a Python Execution Connector
+
+You can build the Python execution connector yourself, or, more commonly, simple install using `pip` (or equivalent).
+
+#### Install
+
+```shell
+    pip install vantiqPythonExecConnector
+```
+
+Once this is done, you can execute the connector by running the command
+
+```
+    vantiqPythonExecConnector
+```
+
+once the configuration file is in place (see [below](#runningConnector)).
+
+#### Build the Connector
+
+To build the connector, clone this [Github Repository](https://github.com/Vantiq/vantiq-extension-sources).
+Once that has been done, run
+
+```
+./gradlew pythonExecSource:clean pythonExecSource:assemble
+```
+
+This will create the distribution in `dist`.
+
+#### <a name="runningConnector" id="runningConnector"></a>Running the Connector
+
+Once you have obtained the connector and created a source in Vantiq,
+you are ready to run the extension.
+
+The underlying sequence of events is as follows:
+
+ - Source is created in Vantiq
+   - When that happens, the source is *up* but not *connected*.
+ - The source extension connects to Vantiq, providing the name of the source on whose behalf it is operating.
+   - At this point, the source is *connected* and operational.
+ - Once connected, the source can be the target of SELECT statements.
+
+The sequence above requires that the connector be told of the Vantiq installation to which to connect, appropriate credentials for doing so, and the name of the source it to perform the `connectExtension` operation.
+
+This information should be provided in a configuration file. 
+(Please read the [Connector SDK's server config documentation](../../extpsdk/README.md#serverConfig) first.)
+Specifically, create a file named `server.config` on a `serverConfig` directory within the working directory in which the connector run.
```

Comment:
`create a file named `server.config`  on/in a `serverConfig` directory within the working directory in which the connector run/runs.`